src/client/cli/cmd/launch.cpp

```diff
@@ -293,7 +293,7 @@ mp::ReturnCode cmd::Launch::request_launch()
             {LaunchProgress_ProgressTypes_INITRD, "Retrieving initrd image: "},
             {LaunchProgress_ProgressTypes_EXTRACT, "Extracting image: "},
             {LaunchProgress_ProgressTypes_VERIFY, "Verifying image: "},
-            {LaunchProgress_ProgressTypes_WAITING, "Preparing image: "}};
+            {LaunchProgress_ProgressTypes_WAITING, "Downloading and preparing image: "}};
```

Comment:
Because the cli design doc is sorely out of date and doesn't take into account new features and behaviors.  I thought it would be more informative that a download and preparation of the image _could_ be taking place in the background while the client is waiting.  Saying "download" indicates to me that it may take a while to complete.

Reply:
OK, but so far we kept informing the user of a single ongoing action, so why change that now? Do we need a new `LaunchProgress_ProgressTypes_UPDATING` or similar instead?

Reply:
Well, really, downloading and preparing are operations that occur serially.  You won't have one without the other.  So being strict about the wording, the longest operation is the download, so maybe this should just say `Downloading image: ` and the prepare part of this will be hidden behind the `Downloading` message?  It seems like a bit much work to split out wording based on a serial operation that can be said in 4 words on the same line to describe the same thing.

Reply:
Ok, I just reverted this change since, after thinking about it, it doesn't even affect what is going on with this PR.

Reply:
For the record, after discussion in IRC, no download happens in this case, so change reverted.